src/sentry/dynamic_sampling/rules/biases/boost_latest_releases_bias.py

```diff
@@ -12,19 +12,23 @@
 )
 from sentry.dynamic_sampling.rules.helpers.latest_releases import ProjectBoostedReleases
 from sentry.dynamic_sampling.rules.utils import (
-    RELEASE_BOOST_FACTOR,
+    BOOST_LATEST_RELEASES_DECAYED_FACTOR,
+    BOOST_LATEST_RELEASES_FACTOR,
```

Comment:
i don't understand the purpose of this rename?

```
RELEASE_BOOST_FACTOR,
```
sounds IMO correct, wdyt @RaduW ?

Reply:
It's aligned with the file naming, this is the rationale.